components/Layout.js

```diff
@@ -1,94 +1,59 @@
-import { Box, theme } from 'ooni-components'
 import PropTypes from 'prop-types'
-import React, { useMemo } from 'react'
-import { ThemeProvider, createGlobalStyle } from 'styled-components'
+import { useMemo } from 'react'
 
-import { StyledStickyNavBar, StyledStickySubMenu } from 'components/SharedStyledComponents'
-import { getDirection } from 'components/withIntl'
+import { StyledStickyNavBar } from 'components/SharedStyledComponents'
+// import { getDirection } from 'components/withIntl'
```

Comment:
Maybe if we don't need this we can just remove the commented out code or mention why it's commented out